share/transformers/mods.py

```diff
@@ -240,7 +240,8 @@ class MODSCreativeWork(Parser):
                     lambda obj: 'invalid' not in obj,
                     tools.Concat(
                         tools.Try(ctx['mods:identifier']),
-                        tools.Try(ctx.header['identifier'])
+                        tools.Try(ctx.header['identifier']),
+                        tools.Try(ctx['mods:location']['mods:url']),
```

Comment:
@aaxelb, do you want to pull the location url stuff into another PR since this one is on hold and also resolves https://openscience.atlassian.net/browse/SHARE-861?